### ISSUE TITLE
No option to carry anything in VS 1.19.0-pre5 w/ carryon 1.7.0-pre1.

### ISSUE THREAD

**Issue by ParkerHGordon** [NerdScurvy/CarryOn#14]
The option doesn't seem to show up at all for me, here's what log events I found from carryon:
```
7.11.2023 21:54:31 [Notification] Mods, sorted by dependency: game, carryon, creative, survival
7.11.2023 21:54:31 [Error] [carryon] An exception was thrown when trying to load assembly:
7.11.2023 21:54:31 [Error] [carryon] Exception: Assembly with same name is already loaded
   at System.Runtime.Loader.AssemblyLoadContext.<LoadFromPath>g____PInvoke|5_0(IntPtr ptrNativeAssemblyBinder, UInt16* ilPath, UInt16* niPath, ObjectHandleOnStack retAssembly)
   at System.Runtime.Loader.AssemblyLoadContext.LoadFromAssemblyPath(String assemblyPath)
   at System.Reflection.Assembly.LoadFrom(String assemblyFile)
   at Vintagestory.Common.ModContainer.<>c__DisplayClass35_0.<LoadAssembly>b__0(String path) in VintagestoryLib\Common\API\ModContainer.cs:line 450
   at System.Linq.Enumerable.SelectListIterator`2.MoveNext()
   at System.Linq.Enumerable.WhereEnumerableIterator`1.ToList()
   at Vintagestory.Common.ModContainer.LoadAssembly(ModCompilationContext compilationContext, ModAssemblyLoader loader) in VintagestoryLib\Common\API\ModContainer.cs:line 476

...

7.11.2023 21:54:42 [Error] [carryon] An exception was thrown when trying to load assembly:
7.11.2023 21:54:42 [Error] [carryon] Exception: Assembly with same name is already loaded
   at System.Runtime.Loader.AssemblyLoadContext.<LoadFromPath>g____PInvoke|5_0(IntPtr ptrNativeAssemblyBinder, UInt16* ilPath, UInt16* niPath, ObjectHandleOnStack retAssembly)
   at System.Runtime.Loader.AssemblyLoadContext.LoadFromAssemblyPath(String assemblyPath)
   at System.Reflection.Assembly.LoadFrom(String assemblyFile)
   at Vintagestory.Common.ModContainer.<>c__DisplayClass35_0.<LoadAssembly>b__0(String path) in VintagestoryLib\Common\API\ModContainer.cs:line 450
   at System.Linq.Enumerable.SelectListIterator`2.MoveNext()
   at System.Linq.Enumerable.WhereEnumerableIterator`1.ToList()
   at Vintagestory.Common.ModContainer.LoadAssembly(ModCompilationContext compilationContext, ModAssemblyLoader loader) in VintagestoryLib\Common\API\ModContainer.cs:line 476
```

Normally I probably wouldn't have made an issue for this in github since carryon pre1 is for VS pre4, but I noticed that another user made an issue where they seem to have been able to at least pick up something, so idk what's wrong with it on my computer. Windows 10, using the same settings files from the last version of carryon. Only enabled non-vanilla mod was carryon. Tried on a new file in both survival and creative.

**Comment by ParkerHGordon** [comment]
Update: it randomly showed up, not sure exactly when. I loaded up my world, started some chiseling, and then I noticed the option to carry baskets was there. I looked at the event logs and noticed a "started Carryon mod" log that I don't think was there yesterday (at least, I didn't see it). So idk if it just has a chance of failing, or if I needed to put something in my offhand and then take it out, or what, but seems to be working now
